docs/site/Loopback-component-authorization.md

```diff
@@ -99,8 +99,8 @@ class.
     defaultDecision: AuthorizationDecision.DENY,
   };
 
-  const binding = app.component(AuthorizationComponent);
-  app.configure(binding.key).to(options);
+  app.component(AuthorizationComponent);
+  app.configure(AuthorizationBindings.COMPONENT).to(options);
```

Comment:
Are you sure that `configure` after `component` works? On other places we have to do `configure` before `component` to make it working?

Reply:
@derdeka I wasn't sure, so I verified with app in https://github.com/strongloop/loopback-next/pull/4571, you can read the PR description to see my testing result.

`this.configure()` is called in the app constructor, and whenever the authorization interceptor is invoked, the options will be injected. So I believe after app runs, making API calls will inject the configured options.

Reply:
I would like to suggest to switch the order to have it consistent with other docs.

Reply:
@derdeka 👌 switched the order to have the configure before mounting component.